compiler/damlc/daml-prim-src/GHC/Tuple/Check.daml

```diff
@@ -0,0 +1,14 @@
+-- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+{-# LANGUAGE DamlSyntax #-}
```

Comment:
I don't think this pragma is necessary - I'm guessing you carried it over from `GHC.Tuple` but it seems it's not needed there either. For now let's drop this one, we can drop the ones in `GHC.Tuple` and `GHC.Classes` later.

```suggestion
```

Reply:
suggestion committed